apps/src/applab/designMode.js

```diff
@@ -14,11 +14,11 @@ var applabCommands = require('./commands');
 var designMode = module.exports;
 var utils = require('../utils');
 
+var gridUtils = require('./gridUtils');
```

Comment:
Nit: Why the extra line between this require and the others? (in a few files in this PR)